packages/eslint-plugin-material-ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-plugin-material-ui",
-  "version": "4.0.0-alpha.37",
+  "version": "5.0.0-beta.0",
```

Comment:
Aren't we supposed to only modified these versions just before releasing ?

Reply:
I reverted all these version bumps.